publish-cli/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Please remove this file not needed with the mockito uplift

Reply:
removed